scio-avro/src/main/scala/com/spotify/scio/avro/syntax/ScioContextSyntax.scala

```diff
@@ -32,6 +32,30 @@ import scala.language.implicitConversions
 import scala.reflect.ClassTag
 import scala.reflect.runtime.universe._
 
+sealed trait AvroFileImpl[T] {
+  def apply(self: ScioContext)(path: String, schema: Schema): SCollection[T]
+}
+
+sealed trait LowPriorityAvroFileImpl {
```

Comment:
Hi @jto seems that this is not used, `defaultImpl` will not be in scope; is it intended? or should `AvroFileImpl` extend `LowPriorityAvroFileImpl`?

Reply:
Oh. Good catch. How are the tests compiling ? oO